csharp/src/Ice/ITransceiver.cs

```diff
@@ -27,8 +27,8 @@ public interface ITransceiver
         /// connections.</param>
         /// <returns>A new connection to the given endpoint.</returns>
         public Connection CreateConnection(
+            IConnectionFactory factory,
```

Comment:
It's odd to have a `CreateConnection` method that takes a connection factory parameter. You would expect the factory to create the connections. It's also unclear what "parent" means here.

Reply:
I've renamed IConnectionFactory to IConnectionManager and the comment for the manager parameter is now `/// <param name="manager">The connection manager which owns the connection.</param>`